working_directory/.gitignore

```diff
@@ -0,0 +1,6 @@
+*
+!.gitignore
+
+# Trick for keeping empty folder in repository courtesy of
+# https://stackoverflow.com/questions/4250063/how-to-gitignore-all-files-folder-in-a-folder-but-not-the-folder-itself
```

Comment:
Why do we need to have an empty folder on GH? Would it be better to have the user create one when needed (similar to how most C code compilations require you to create a `build` dir) and have the whole thing gitignored? Could maybe include a check at the beginning of the script to see if the directory exists and if not raise an error

Reply:
Yeah, that's totally fair. This was just easier than writing the few lines to store an empty folder

Reply:
This is a style thing but I don't think that the empty folder approach is entirely without merit. We're not dealing with a super complicated directory so just having an empty folder in place makes it easier on the user and more idiot-proof. We can go without here but that's my five cents on the matter.